cli/reactjs_jsx_ppx.ml

```diff
@@ -2054,7 +2061,7 @@ module V4 = struct
         let retPropsType =
           Typ.constr ~loc:pstr_loc
             (Location.mkloc (Lident "props") pstr_loc)
-            (makePropsTypeParams namedTypeList)
+            (makePropsTypeExternal namedTypeList)
```

Comment:
Why does an external definition need special treatment?

Reply:
Wow, you're a quick reviewer. Actually, it should be the same as the `makePropsTypeExternal`, so I'm just about to merge and rename those functions.